<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btnDrogForm</class>
 <widget class="QWidget" name="btnDrogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>btnDrogForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
 border-left: 0px solid #00aa00;
 border-right: 0px solid #00aa00;
 border-top: 0px solid #00aa00;
 border-bottom: 0px solid #00aa00;
  background-color: rgb(111, 111, 111);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame #frame{
 border-left: 0px solid #00aa00;
 border-right: 0px solid #00aa00;
 border-top: 0px solid #00aa00;
 border-bottom: 0px solid #00aa00;
 background-color: rgb(111, 111, 111);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="toolTipDuration">
             <number>1</number>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#1296db;</string>
            </property>
            <property name="text">
             <string>标注</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="indent">
             <number>-6</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnMouse">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>选择</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#btnMouse{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#btnMouse:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#btnMouse:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}</string>
            </property>
            <property name="text">
             <string>选择</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnDrawLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>画笔</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#btnDrawLine{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#btnDrawLine:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#btnDrawLine:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}</string>
            </property>
            <property name="text">
             <string>画笔</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnEras">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>擦除</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#btnEras{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#btnEras:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#btnEras:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}</string>
            </property>
            <property name="text">
             <string>擦除</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnClear">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>清除</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#btnClear{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#btnClear:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#btnClear:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}</string>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnApply">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>应用</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#btnApply{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#btnApply:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#btnApply:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}</string>
            </property>
            <property name="text">
             <string>应用</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_menubar">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_menu">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>多类别</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton#toolButton_menu{
color: rgb(255, 255, 255);
border-radius:2px;
background:transparent;
border: 1px solid #1296db;
}
QToolButton#toolButton_menu:checked {  
    /* 改变背景色 */    
	background-color: rgb(0, 170, 255);
}
QToolButton#toolButton_menu:hover {
background-color: rgb(0, 170, 255);
}
QToolTip{
border: 0px solid black;
background-color:  rgb(74, 74, 74);
color:white
}
QToolButton#toolButton_menu:menu-indicator{
image:none
}</string>
            </property>
            <property name="text">
             <string>类别</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_20">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="minimumSize">
             <size>
              <width>46</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>46</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(111, 111, 111);
border-radius:2px;
border: 1px solid #1296db;
background:transparent;
QSpinBox::up-button
{
	width:0px;
}</string>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_15">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 0);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_16">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(74, 74, 74);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>7</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:2px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_min">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:2px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_18">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
